src/SFA.DAS.LevyTransferMatching.Web/Views/Pledges/Pledges.cshtml

```diff
@@ -19,6 +19,24 @@
 
 <main class="govuk-main-wrapper govuk-main-wrapper--auto-spacing" id="main-content">
     <div class="govuk-width-container">
+
+        @if (Model.PledgeClosedShowBanner)
+        {
+            <div class="govuk-notification-banner govuk-notification-banner--success" role="alert" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+                <div class="govuk-notification-banner__header">
+                    <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+                        Success
+                    </h2>
+                </div>
+                <div class="govuk-notification-banner__content">
+                    <h3 class="govuk-notification-banner__heading">
+                        Transfer pledge closed
+                    </h3>
+                    <p class="govuk-body">You closed the transfer pledge @Model.PledgeClosedEncodedPledgeId.</p>
```

Comment:
Pedantic, but does this property require the "PledgeClosed" prefix? It reads a little odd to me.

Reply:
This field not required any more and removed.